src/main/java/com/cupid/jikting/jwt/filter/JwtAuthenticationProcessingFilter.java

```diff
@@ -53,6 +60,12 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         saveAccessTokenAuthentication(request, response, filterChain);
     }
 
+    private boolean isNotRequiredJwtAuthentication(HttpServletRequest request) {
+        return tokenAUthorizationWhiteList.stream()
+                .anyMatch(uri -> request.getRequestURI().contains(uri))
+                || (request.getRequestURI().contains(signUpUri) && request.getMethod().equals(HttpMethod.POST.name()));
```

Comment:
p3. stream에서 개행을 지우는건 어떨까요?

```suggestion
        return tokenAUthorizationWhiteList.stream().anyMatch(uri -> request.getRequestURI().contains(uri))
                || (request.getRequestURI().contains(signUpUri) && request.getMethod().equals(HttpMethod.POST.name()));
```